lib/screens/tree_dict/dict_builder.dart

```diff
@@ -0,0 +1,13 @@
+import 'package:flutter/material.dart';
+import 'package:lorax/jsonModels/Data.dart';
+
+
+class DictBuilder extends StatelessWidget {
+  final List<Data> data;
```

Comment:
Is this file for list view?

Reply:
I created it for making the builder for ListView. But went with doing it in the plictionary.dart file

Reply:
then what is the use of this file,

if no use then delete that

Reply:
I was planning to add the builder function in this class once it started working so that it will be more organized. If still not required I will delete.

Reply:
ok, fine leave it